resources/apache/README.md

```diff
@@ -33,4 +33,6 @@ Each Apache instance generate ~25 time series.
 For further information, consult [Apache documentation](https://httpd.apache.org/).
 
 # Attributions
-Configuration files and dashboards maintained by [Sysdig team](https://sysdig.com/).
\ No newline at end of file
+Configuration files and dashboards maintained by [Sysdig team](https://sysdig.com/).
+
```

Comment:
The configuration used here is based in the one published in this [blog post of Robust Perception](https://www.robustperception.io/getting-metrics-from-apache-logs-using-the-grok-exporter).